IntegrationTests/AiGeneratorTests.cs

```diff
@@ -22,122 +22,122 @@ public AiGeneratorTests(Fixture fixture)
     [Fact]
     public void ImageGenerationsResponse_GenerationRequestIdReturned()
     {
-        _fixture.ImageGenerationsResponse.GenerationRequestId.Should().NotBeNullOrEmpty();
+        _fixture.ImageGenerationsResponse?.GenerationRequestId.Should().NotBeNullOrEmpty();
```

Comment:
I think you have to check this for null. Otherwise, the assertion will get skipped when it is null and the test will pass when it shouldn't.

Reply:
Disagree.  It should either a) throw a `NullReferenceException` when trying to get the `GenerationRequestId` property or b) it'll fail the test since it'll be `null`.  But I can test that for sure real quick ...

Reply:
OK so I've been doing it wrong™️ for awhile then.  You're right.  WTF.

``` csharp
public class SomeTests
{
    private SomeClass someNullThing = null;

    [Fact]
    public void MarkIsRight()
    {
        someNullThing?.TheValue.Should().NotBeNullOrEmpty();
    }

    [Fact]
    public void YetThisFails()
    {
        var theValue = someNullThing?.TheValue;
        theValue.Should().NotBeNullOrEmpty();
    }

    public class SomeClass
    {
        public string TheValue { get; set; } = "HasValue";
    }
}
```

Reply:
I turned off `nullable` and removed all these.  We'll get `NullReferenceExceptions` now when those aren't set.